Autopilot/Src/MPXV7002DP.cpp

```diff
@@ -0,0 +1,65 @@
+#include "airspeed.hpp"
+#include "adc.h"
+#include <stddef.h>
+#include <cmath>
+
+#define VS 5
+#define DENSITY_AIR 1.204
+#define ADC_RESOLUTION 4096
+
+
+MPXV7002DP::MPXV7002DP() {
+    //constructor
+    MX_ADC2_Init();
```

Comment:
A quick look through adc.c shows me adc3 is used for airspeed. adc 2 is actually used for battery stuff. So you'll have to use adc3 in your driver.